index.html

```diff
@@ -72,8 +70,105 @@ <h3 class="ability_header">Skills</h3>
+                    <h3>Multi-Post Stories</h3>
+                    <p class="work_info">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul class="languages">
+                        <li>css</li>
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button type="button" class="btn">See Project</button>
+                </div>
+            </div>
+
+            <div class="work_four">
+                <img src="placeholder.png" alt="image placeholder">
+                <div class="supporting_text">
+                    <h3>Multi-Post Stories</h3>
+                    <p class="work_info">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul class="languages">
+                        <li>css</li>
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button type="button" class="btn">See Project</button>
+                </div>
+            </div>
+            </section>
+            <section>
+                <div class="contact_us">
+                <h2>CONTACT US</h2>
+                <p class="work_info">
+                    If you have an application you are interested in developing,
+                    a feature that you need built or a project that needs coding. 
+                    I'd love to help with it 
+                </p>
+                <form name="contact_form" class="contact_form">
+                    <input type="text" placeholder="Text Input" class="name">
+                    <input type="email" placeholder="stanleyjones@gmail.com" class="email">
+                    <input type="text" placeholder="Enter text here" class="customer_info">
+                </form>
+                </div>
+                <div class="btn_links">
+                    <button type="button" class="btn">See Project</button>
+                    <ul class="social_media_links">
+                        <li class="left_border"><i class="fa-brands fa-github"></i></li>
+                        <li><i class="fa-brands fa-linkedin"></i></li>
+                        <li><i class="fa-brands fa-twitter"></i></li>
+                        <li><i class="fa-brands fa-angellist"></i></li>
+                    </ul>
+                    <p>baronjones976@gmail.com</p>
+                </div>
```

Comment:
- [ ] [ OPTIONAL ] kindly remove this contact section is not part of the requirement  to add it, you will implement it in the next project, here you will implement only card section :grinning: :grinning:

Reply:
kindly note that the email section you are asking is at the end of the contact page you want me to remove